code/lib/cli/src/initiate.ts

```diff
@@ -227,9 +227,15 @@ const projectTypeInquirer = async (
         })),
       },
     ]);
-    return installStorybook(frameworkAnswer.manualFramework, packageManager, options);
+
+    if (manualFramework) {
+      return installStorybook(manualFramework, packageManager, options);
+    }
   }
-  return Promise.resolve();
+
+  logger.log();
+  logger.log('For more information about installing Storybook: https://storybook.js.org/docs');
+  process.exit(0);
```

Comment:
@shilman reminder to check this later, thanks!

Reply:
I'm ok with this. I think it's not great for our telemetry, but hopefully it's a small corner case.